packages/autocertifier/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@streamr/autocertifier",
+  "version": "0.0.1-tatum.6",
+  "description": "Automatic TSL Certificate Manager for Streamr Network",
```

Comment:
What it means that this is automatic manager? Would it make sense to describe it as a server which provides self-signed TLS certificates

Reply:
The certificates are proper as we use google service. If the server would provide self-signed certificates then browser nodes would not be able to connect to autocertified nodes